index.html

```diff
@@ -98,31 +91,18 @@ <h1 class="logo me-auto me-lg-0">
             <li><a class="nav-link scrollto" href="#about">About</a></li>
             <li><a class="nav-link scrollto" href="#menu">Menu</a></li>
             <li><a class="nav-link scrollto" href="#specials">Specials</a></li>
-            <li><a class="nav-link scrollto" href="#events">Events</a></li>
-            <li><a class="nav-link scrollto" href="#chefs">Chefs</a></li>
             <li><a class="nav-link scrollto" href="#gallery">Gallery</a></li>
             <li class="dropdown">
               <a href="#"
-                ><span>Drop Down</span> <i class="bi bi-chevron-down"></i
+                ><span>Other items</span> <i class="bi bi-chevron-down"></i
```

Comment:
We need to think of another name, Other items doesn't look nice. Ask the client.

Reply:
i changed to sunday